patroni/ctl.py

```diff
@@ -846,6 +1290,24 @@ def _do_failover_or_switchover(obj: Dict[str, Any], action: str, cluster_name: s
 @click.pass_obj
 def failover(obj: Dict[str, Any], cluster_name: str, group: Optional[int],
              leader: Optional[str], candidate: Optional[str], force: bool) -> None:
+    """Process ``failover`` command of ``patronictl`` utility.
+
+    Perform a failover operation immediately in the cluster.
+
+    .. note::
+        If *leader* is given perform a switchover instead of a failover.
+
+    .. seealso::
+        Refer to :func:`_do_failover_or_switchover` for details.
+
+    :param obj: Patroni configuration.
+    :param cluster_name: name of the Patroni cluster.
+    :param group: filter Citus group within we should perform a failover or switchover. If ``None`` consider members of
+        the coordinator group.
```

Comment:
The same as https://github.com/zalando/patroni/pull/2687/files#diff-9dada8f509128404d7027246ac9622d2484fc77758c6a4af9dc7cc8c081d9fffR1160